mak/COPY

```diff
@@ -93,6 +108,7 @@ COPY=\
 	$(IMPDIR)\core\sys\osx\mach\thread_act.d \
 	\
 	$(IMPDIR)\core\sys\osx\sys\cdefs.d \
+	$(IMPDIR)\core\sys\osx\sys\event.d \
```

Comment:
Wait, did you mean just remove the copy commands for core/sys/osx/sys/event.d from make files or also remove the file core/sys/osx/sys/event.d?